benchmark/cublas/blas1/asum.cpp

```diff
@@ -0,0 +1,156 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2016 Codeplay Software Limited
```

Comment:
Could you please remove the year from the license in all of the files? We had recently decided to remove the year from all of the licenses in SYCL-BLAS/DNN but I guess there are still files which have these years.